### PR TITLE
add singly charged Scalar GMmodel

### PR DESCRIPTION
**Pull request by jsalfeld**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_proc_card.dat]: remove this import as is overridden by the import below.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                     
+ 263000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: you are using 5F PDF but define the proton as 4F (as per mg5_amc@nlo default) which is inconsistent. ask the theorists who provide the model which PDF scheme should be used

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_customizecards.dat [hunk 3]
```diff
@@ -0,0 +1,5 @@
+set param_card WH5pp 78.925
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_customizecards.dat]: pardon my ignorance, where is the M1000 defined?

Comment (jsalfeld) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_customizecards.dat]: it is computed from the parameters defined in the paramcard.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat (hunk 2 above)
Comment (jsalfeld) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: yes bugger, will change to 263400, 4F is default in the run_cards of the model. Also checking with theorists

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_proc_card.dat (hunk 1 above)
Comment (jsalfeld) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_proc_card.dat]: will do

**Comment by jsalfeld** [comment]
Done... can you review asap?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat [hunk 4]
```diff
@@ -277,5 +277,5 @@
 30 50 = sys_matchscale # variation of merging scale
 # PDF sets and number of members (0 or none for all members).
 # CT10nlo.LHgrid = sys_pdf # matching scales
-NNPDF30_lo_as_0130.LHgrid = sys_pdf
+CT10nlo.LHgrid = sys_pdf
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: not vital, but this is also a 5F PDF. Do you have a specific reason to keep it as systematics?

Comment (jsalfeld) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: yes I noticed just now, I followed that part what was in the default run_card

**Comment by perrozzi** [comment]
apart from the comment on syst PDF, looks ok to me

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat (hunk 4 above)
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: if you don't plan to use alternative pdf sets can probably stay, or remove it. up to you

Comment (jsalfeld) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: yes, not planning to use different pdf sets, just weight variations of the default one. This does not interfere right?

Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat]: to my knowledge is just an additional set of weights stored

**Pull request merged**